cardano-db-sync/src/Cardano/DbSync.hs

```diff
@@ -441,6 +441,8 @@ chainSyncClient trce metrics latestPoints currentTip actionQueue =
 logProtocolMagic :: Trace IO Text -> Crypto.ProtocolMagic -> IO ()
 logProtocolMagic tracer pm =
   liftIO . logInfo tracer $ mconcat
-    [ "NetworkMagic: ", textShow (Crypto.getRequiresNetworkMagic pm), " "
+    [ "NetworkMagic: "
+    , textShow (Crypto.getRequiresNetworkMagic pm)
+    , " "
```

Comment:
Why? That change is purely whitespace?

Reply:
Yes, this is my preference. I see this as more clear to read.

Reply:
I see these extra new lines as un-necessary. That is my preference.